### ISSUE TITLE
cant launch the game

### ISSUE THREAD

**Issue by heckrum** [sp614x/optifine#5370]
## Description of Issue
ive been playinng minecraft for a while now with optifine installed, and today i tried to launch minecraft when it just crashed me launcher. i tried again, crashed. again, crashed. i tried it with the latest version, and it opened. i try to do it with optifine, crashed. i deleted and reinstalled optifine, still nothing.

## Steps to Reproduce
1. i dunno

## OptiFine Version
1.16.5
## Installation Method
i went to the optifine site and downloaded it normally.

## OptiFabric Version
i dont know what that is

## Log Files/Crash Reports
i dont have any for this

## Prior Testing
it does not happen in vanilla

in conclusion, idk its a bug with optifine thats never happened before. i really dont like it, please fix it.

**Comment by MrKrakenied** [comment]
#5368

**Comment by heckrum** [comment]
this tells me literally nothing dude

**Comment by Cart3r1234** [comment]
disable the beta version of the launcher, that is causing the crash.

**Comment by heckrum** [comment]
yeah that worked

**Comment by heckrum** [comment]
thanks

**Comment by KaiAF** [comment]
Closing this as it's been fixed by Mojang
